cypress/page/shipping-step.page.ts

```diff
@@ -3,15 +3,15 @@ class ShippingStepPage {
     private termsCheck: string;
 
     constructor() {
-        this.checkbtn = ".cart_navigation > .button > span";
+        this.checkbtn = ".cart_navigation .button span";
```

Comment:
Find a better one, in this case you can guide with the name of the button for example